.github/workflows/recurring-release-dev.yml

```diff
@@ -0,0 +1,24 @@
+name: "Recurring release: Dev"
+on:
+  # Can be manually triggered
+  workflow_dispatch:
+  # Runs 5:05 am EDT Monday to Friday.
```

Comment:
Changed to run once nightly. The timing is 5:05 am to allow any rebuild of the Tugboat content source to be complete before trying to run.

Reply:
5:35 am for staging; see below.